docs/overview/automatic.rst

```diff
@@ -91,7 +91,7 @@ file and there is user's option:
     command_to_install(Bar)
   endif()
 
-In case it's not CMake code this will look even fancy, say ``requirements.json``:
+Or, in the case that it isn't CMake code, this might by something fancy like ``requirements.json``:
```

Comment:
Spell check doesn't like `isn't`. This is kind of a spell check limitation, you can add `isn` to `spelling.txt` or use `is not` here.

Reply:
I'm afraid no, at least that's what I remember.

Reply:
Yip, it was still grumpy.